zebra-state/src/service/finalized_state/disk_db.rs

```diff
@@ -285,10 +285,17 @@ impl DiskDb {
     /// Returns the database options for the finalized state database.
     fn options() -> rocksdb::Options {
         let mut opts = rocksdb::Options::default();
+        let mut block_based_opts = rocksdb::BlockBasedOptions::default();
 
         opts.create_if_missing(true);
         opts.create_missing_column_families(true);
 
+        // Use the recommended Ribbon filter setting for all column families.
+        // (Ribbon filters are faster than Bloom filters in Zebra, as of April 2022.)
+        //
+        // (They aren't needed for single-valued column families, but they don't hurt either.)
+        block_based_opts.set_ribbon_filter(9.9);
```

Comment:
Why 9.9? Most examples seem to use 10.0

Reply:
I was working off:
https://github.com/facebook/rocksdb/wiki/RocksDB-Bloom-Filter#ribbon-filter

(I don't think the difference will matter that much.)